apps/api/tsconfig.json

```diff
@@ -1,18 +1,15 @@
 {
+  "extends": "@recallnet/typescript-config/base.json",
   "compilerOptions": {
-    "target": "ES2020",
-    "module": "commonjs",
-    "lib": ["es2020"],
+    "declaration": false,
+    "declarationMap": false,
     "outDir": "./dist",
     "rootDir": ".",
-    "strict": true,
-    "esModuleInterop": true,
-    "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "resolveJsonModule": true,
     "experimentalDecorators": true,
     "emitDecoratorMetadata": true,
-    "sourceMap": true
+    "sourceMap": true,
+    "noUncheckedIndexedAccess": false
```

Comment:
This is set to `true` in the shared config, which is the better setting I think. I like that when you access an array by index, the result can be `undefined`. Forces you to deal with that case. So, removing this line will cause us to deal with all that, which is better left for a separate PR.